.github/workflows/tests.yml

```diff
@@ -195,22 +195,22 @@ jobs:
     - uses: actions/checkout@v2
     - run: make md-lint
 
-  # Go Releaser
-  release:
-    runs-on: ubuntu-latest
-    name: 'release: snapshot'
-    steps:
-    - uses: actions/setup-go@v2
-      with:
-        go-version: 1.14
-    - uses: actions/checkout@v2
-    - name: Import GPG key
-      id: import_gpg
-      uses: crazy-max/ghaction-import-gpg@v3
-      with:
-        gpg-private-key: "${{ secrets.GPG_PRIVATE_KEY }}"
-        passphrase: "${{ secrets.GPG_PASSPHRASE }}"
-    - run: make release-test
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
+#  # Go Releaser
```

Comment:
Why comment this section?

Reply:
this step is always failed now and it's no need to create a release for each commit

Reply:
it works in release.yml, I think maybe the secrets.GPG_PRIVATE_KEY is not readable in PR CI